plugins/modules/k8s_service.py

```diff
@@ -195,10 +212,29 @@ def argspec():
     return argument_spec
 
 
-def execute_module(module, k8s_ansible_mixin):
-    """ Module execution """
-    k8s_ansible_mixin.set_resource_definitions(module)
+def perform_action(svc, module, definition):
+    result = {}
+    module.warnings = []
+
+    if module.params["validate"] is not None:
+        module.warnings = validate(svc.client, module, definition)
```

Comment:
This module does not support the `validate` param. I was a bit surprised to see that this passed CI since it should throw an `AttributeError` exception. Upon further investigation, I found that there is only a single test for this module and it uses `ignore_errors: yes`. :scream:

Reply:
😭I wanted to replicate some things from the runner... but I didn't pay attention at the naming and the other params... my fault! 
I also noticed `k8s_service` is not fully covered with integration tests.

Reply:
@gravesm Can we open some issues for the modules that are not fully covered by integration tests?

Reply:
Yes, definitely.